tools/provision/formula/librdkafka.rb

```diff
@@ -0,0 +1,38 @@
+class Librdkafka < Formula
+  desc "The Apache Kafka C/C++ library"
+  homepage "https://github.com/edenhill/librdkafka"
+  url "https://github.com/edenhill/librdkafka/archive/v0.9.4.tar.gz"
+  sha256 "5007ad20a6753f709803e72c5f2c09483dcbce0f16b94b17cf677fb3e6045907"
+
+  head "https://github.com/edenhill/librdkafka.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "lzlib"
+  depends_on "openssl"
+
+  def install
+    args = [
+      "--disable-dependency-tracking",
+      "--prefix=#{prefix}",
+      "--enable-static",
+      "--disable-ssl",
```

Comment:
Yeah, it wouldn't build when I with SSL enabled.  Since we did not need that internally, I didn't bother investigating into.  I figured whomever needs to have this feature enabled would deal with it then.  Thoughts?